tests/testthat/testSel3D.R

```diff
@@ -0,0 +1,109 @@
+  #Manually check if top matches on BLAST are present in the output for the right positions
+  result <- SEL3D(someResults, convert=FALSE)
+  expect_equal(length(result$pdb.id), 1)
+  expect_true("3M1N_A" %in% result$pdb.id)
+  #check q.start
+  expect_equal(result[result$pdb.id == "3M1N_A", 8], 25)
+  #check q.end
+  expect_equal(result[result$pdb.id == "3M1N_A", 9], 197)
+  #check s.start
+  expect_equal(result[result$pdb.id == "3M1N_A", 10], 3)
+  #check s.end
+  expect_equal(result[result$pdb.id == "3M1N_A", 11], 175)
+  
+  manyResults <- "MEEPQSDPSVEPPLSQETFSDLWKLLPENNVLSPLPSQAMDDLMLSPDDIEQWFTEDPGPDEAPRMPEAAPPVAPAPAAPTPAAPAPAPSWPLSSSVPSQKTYQGSYGFRLGFLHSGTAKSVTCTYSPALNKMFCQLAKTCPVQLWVDSTPPPGTRVRAMAIYKQSQHMTEVVRRCPHHERCSDSDGLAPPQHLIRVEGNLRVEYLDDRNTFRHSVVVPYEPPEVGSDCTTIHYNYMCNSSCMGGMNRRPILTIITLEDSSGNLLGRNSFEVRVCACPGRDRRTEEENLRKKGEPHHELPPGSTKRALPNNTSSSPQPKKKPLDGEYFTLQIRGRERFEMFRELNEALELKDAQAGKEPGGSRAHSSHLKSKKGQSTSRHKKLMFKTEGPDSD"
+  #Manually check if top matches on BLAST are present in the output for the right positions
+  result <- SEL3D(manyResults, convert=FALSE)
+  expect_equal(length(result$pdb.id), 4)
+  expect_true("4QO1_B" %in% result$pdb.id)
+  expect_equal(result[result$pdb.id == "4QO1_B", 8], 92)
+  expect_equal(result[result$pdb.id == "4QO1_B", 9], 312)
+  expect_equal(result[result$pdb.id == "4QO1_B", 10], 1)
+  expect_equal(result[result$pdb.id == "4QO1_B", 11], 221)
+  expect_equal(result[result$pdb.id == "4QO1_B", 17], "Homo sapiens")
+  
+  expect_true("5HPD_A" %in% result$pdb.id)
+  expect_equal(result[result$pdb.id == "5HPD_A", 8], 2)
+  expect_equal(result[result$pdb.id == "5HPD_A", 9], 61)
+  expect_equal(result[result$pdb.id == "5HPD_A", 10], 99)
+  expect_equal(result[result$pdb.id == "5HPD_A", 11], 158)
+  expect_equal(result[result$pdb.id == "5HPD_A", 17], "Homo sapiens")
+  
+  expect_true("1OLG_A" %in% result$pdb.id)
+  expect_equal(result[result$pdb.id == "1OLG_A", 8], 319)
+  expect_equal(result[result$pdb.id == "1OLG_A", 9], 360)
+  expect_equal(result[result$pdb.id == "1OLG_A", 10], 1)
+  expect_equal(result[result$pdb.id == "1OLG_A", 11], 42)
+  expect_equal(result[result$pdb.id == "1OLG_A", 17], "Homo sapiens")
+  
+  expect_true("1DT7_X" %in% result$pdb.id)
+  expect_equal(result[result$pdb.id == "1DT7_X", 8], 369)
+  expect_equal(result[result$pdb.id == "1DT7_X", 9], 388)
+  expect_equal(result[result$pdb.id == "1DT7_X", 10], 1)
+  expect_equal(result[result$pdb.id == "1DT7_X", 11], 22)
+  expect_equal(result[result$pdb.id == "1DT7_X", 17], "Homo sapiens")
+  
+})
+
+
+test_that("a corrupt input does not lead to corrupted output", {
```

Comment:
Under "parameter errors are correctly handled", you should have tests like

+    # if silent is not given logical input
+    expect_error(importSNV.TCGA(fMAF = testMAF, tOut, silent = 1), ".checkArgs> \"silent\" mode error: argument has mode \"numeric\" but function expects mode \"logical\".")
+    expect_error(importSNV.TCGA(fMAF = testMAF, tOut, silent = NULL), ".checkArgs> \"silent\" mode error: argument has mode \"NULL\" but function expects mode \"logical\".")
+    expect_error(importSNV.TCGA(fMAF = testMAF, tOut, silent = c(TRUE, TRUE)), ".checkArgs> \"silent\" length error: argument has length 2 but function expects length 1.")